media-video/obs-studio/obs-studio-25.0.8-r1.ebuild

```diff
@@ -73,15 +78,37 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
+QA_PREBUILT="
+	/usr/lib*/obs-plugins/swiftshader/libEGL.so
+	/usr/lib*/obs-plugins/swiftshader/libGLESv2.so
+	/usr/lib*/obs-plugins/libEGL.so
+	/usr/lib*/obs-plugins/chrome-sandbox
+	/usr/lib*/obs-plugins/libcef.so
+	/usr/lib*/obs-plugins/libGLESv2.so
+"
+
 PATCHES=( "${FILESDIR}/${PN}-25.0.8-gcc-10-build.patch" )
 
 pkg_setup() {
 	use python && python-single-r1_pkg_setup
 }
 
+src_unpack() {
+	default
+	if [ "${PV}" != "9999" ]; then
```

Comment:
I'd prefer '[[' instead of '[', but I can change that later while changing the existing ones, so this is more for information than anything.  :]

Reply:
I've changed it to [[ to match the condition at the top of the file